tests/ui/logout.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "playwright/test";
+
+
+test("logout", async ({ page, baseURL }) => {
+  await page.goto(`/objekty`);
+
+  await page.locator(".right.menu .account-dropdown").click();
+  await page.waitForSelector(".menu.transition");
+
+  await page.evaluate(() => {
+    const links = Array.from(
+      document.querySelectorAll(".menu.transition a.item")
+    );
+    const logoutLink = links.find(
+      (link) => link.textContent.trim() === "Odhlášení"
```

Comment:
A terrible selector, what if the string later changes to "Odhlasit se", or there will be a request to provide english translation